.gitignore

```diff
@@ -5,4 +5,14 @@ src/config/
 *.css.map
 .DS_Store
 .vscode/
-.idea/
\ No newline at end of file
+.idea/
+
+# https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
+# We currently aren't using Zero-Installs
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
```

Comment:
All of the files in .yarn/sdks is auto-generated, but we're instructed to commit those files to SCM